Example/ExampleTests/UICollectionViewTests.swift

```diff
@@ -70,18 +74,38 @@ class UICollectionViewTests: XCTestCase {
         var resultIndexPath: IndexPath? = nil
         var resultCollectioViewCell: UICollectionViewCell? = nil
 
-        subscription = collectionView.didEndDisplayingCellPublisher
+        collectionView.didEndDisplayingCellPublisher
             .sink(receiveValue: { cell, indexPath in
                 resultCollectioViewCell = cell
                 resultIndexPath = indexPath
             })
+            .store(in: &subscriptions)
 
         let givenIndexPath = IndexPath(row: 1, section: 0)
         let givenCollectionViewCell = UICollectionViewCell()
         collectionView.delegate!.collectionView?(collectionView, didEndDisplaying: givenCollectionViewCell, forItemAt: givenIndexPath)
 
         XCTAssertEqual(resultIndexPath, givenIndexPath)
         XCTAssertEqual(resultCollectioViewCell, givenCollectionViewCell)
-        subscription.cancel()
+    }
+    
+    func test_didSelectItemAt_for_multiple_subscribers() {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+
+        var resultIndexPaths = [IndexPath]()
+
+        collectionView.didSelectItemPublisher
+            .sink(receiveValue: { resultIndexPaths.append($0) })
+            .store(in: &subscriptions)
+
+        collectionView.didSelectItemPublisher
+            .sink(receiveValue: { resultIndexPaths.append($0) })
+            .store(in: &subscriptions)
```

Comment:
Preferably store to two different arrays and make sure they're equal, would be a more realistic test